{{cookiecutter.repo_name}}/.github/workflows/{{cookiecutter._ci_name}}.yaml

```diff
@@ -27,7 +27,6 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       stable-python-version: {{ "${{ steps.get-compatible-python.outputs.stable-python }}" }}
-      python-matrix: {{ "${{ steps.get-compatible-python.outputs.python-versions }}" }}
```

Comment:
Should we just drop this altogether and set the other jobs using "stable python" to just 3.11?

I think we're at a point where we expect folks to update their workflows with new python versions, so removing this complexity might be better?

Reply:
Sure, makes sense to me.